src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -76,6 +76,8 @@ private void InitializeDelegateCommands()
             SetNumberFormatCommand = new DelegateCommand(SetNumberFormat, CanSetNumberFormat);
             GetBranchVisualizationCommand = new DelegateCommand(GetBranchVisualization, CanGetBranchVisualization);
             DumpLibraryToXmlCommand = new DelegateCommand(model.DumpLibraryToXml, model.CanDumpLibraryToXml);
+            ShowGalleryCommand = new DelegateCommand(ShowGallery, o => true);
+            CloseGalleryCommand = new DelegateCommand(CloseGallery, o => true);
```

Comment:
Why don't we do `p => OnRequestShowGallery()` and `p => OnRequestCloseGallery()` here? It can save two methods.